pubber/src/main/java/daq/pubber/ManagerBase.java

```diff
@@ -80,10 +80,16 @@ public void error(String message) {
   protected void updateInterval(Integer sampleRateSec) {
     int reportInterval = ofNullable(sampleRateSec).orElse(DEFAULT_REPORT_SEC);
     int intervalSec = ofNullable(options.fixedSampleRate).orElse(reportInterval);
+    if (intervalSec < DISABLED_INTERVAL) {
+      error(format("Dropping update interval, negative sample rate %ds", intervalSec));
```

Comment:
sample rate doesn't necessarily need to be negative, just less than DISABLED_INTERVAL?  Maybe adjust error message accordingly.